### ISSUE TITLE
i hate your religion

### ISSUE THREAD

**Issue by Rayista** [hakandundar34coding/system-monitoring-center#125]
So I'm going to eat pork kebabs at your door now

**Comment by arnavbhatt288** [comment]
@Rayista Then prepare to get stoned for free and not in a good way.

**Comment by arnavbhatt288** [comment]
@Rayista Changing your comments will not help now, prepare to get stoned. :rock:

**Comment by Rayista** [comment]
> @Rayista Changing your comments will not help now, prepare to get stoned. 🪨

You are not the OP, you will not understand.

**Comment by arnavbhatt288** [comment]
> > @Rayista Changing your comments will not help now, prepare to get stoned. rock
> 
> You are not the OP, you will not understand.

And you will never understand the concept of a religion as you are a degenerate.

**Comment by AmashiroNoah** [comment]
It's a better way to block the project author or report abuse, rather than offending other's religion for https://github.com/hakandundar34coding/system-monitoring-center/pull/123. Anyway, offending other's religion is not a good way to solve problems but a way to ruin open source community.

**Comment by AmashiroNoah** [comment]
感觉如果你不删贴的话，只能把这里当成原批笑话看了，并且最后显得我们维权的手段很低能，，，
在这里发帖不如去 #123 发泄 @Rayista

**Comment by JaneValleyhasBtaken** [comment]
let's get stoned together! I hear that'll be full of joy🥰

**Comment by njlyf2011** [comment]
Why solve the problem in such a naive way?

**Comment by njlyf2011** [comment]
I  think it's not right to express dissatisfaction in this way.  I suggest you apologize.

**Comment by Rayista** [comment]
Forget it, the poor Muslims who can't afford to eat pork are about to follow Allah and blow themselves up.

**Comment by Rayista** [comment]
> > > @Rayista Changing your comments will not help now, prepare to get stoned. rock
> > 
> > 
> > You are not the OP, you will not understand.
> 
> And you will never understand the concept of a religion as you are a degenerate. Hell is awaiting for you.

You're right, we should go to a Turkish mosque and eat pork kebabs with Allah.
